### ISSUE TITLE
basePath has to be set somehow when running in CLI mode

### ISSUE THREAD

**Issue by cbleek** [yawik/JobsByMail#2]
In our demo I've set the server-url inluding our basepath as: `--server-url=https://yawik.org/demo/` 

The links the jobposting and the images are broken because the basepath is missing.

The Mail from the demo contains the following HTML

```
 <h2><a href="https://yawik.org/de/jobs/view?subscriberUri=https://yawik.org/subscriber/1&id=58f77cdf4e197fc54be3fed7">Test Anzeigen Ersteller (m/w)</a></h2>
            <div><img src="https://yawik.org/static/Organizations/Image/8/d/58f8be664e197f2e1ee3fed8.jpg" height="80"></div>
                Common        <p>Frankfurt am Main</p>

```
@fedis: Can you take a look at this?

**Comment by fedys** [comment]
I see, there is a missing `/demo` part. I can look at it tomorrow.

**Comment by fedys** [comment]
Fixed via 8ff9a78c2b23cd741dae3ed0a37b1cfc7dbf3782

**Comment by cbleek** [comment]
perfect! It's working. Thank you.
